### DISCUSSION TITLE
“Horizontal rule renderer has been deprecated”?

### DISCUSSION THREAD

**Discussion by sorsted** [Witiko/markdown#370] [Q&A]
Whenever I use a horizontal rule `----------`, I get a warning saying that this renderer has been deprecated and will be removed in Markdown 3.0.0. Why is that the case? I like to use this to make large breaks in the text, by defining e.g.
```latex
\def\markdownRendererHorizontalRule{\fancybreak{*\quad*\quad*}}
```
where `\fancybreak` is a command from `memoir`. I wouldn’t want this functionality to be removed.

**Comment by Witiko** [comment]
In https://github.com/Witiko/markdown/pull/201, the horizontal rule renderer has been renamed to thematic break to describe the semantics of the element rather than how it is commonly rendered. You can adapt your code by replacing `\markdownRendererHorizontalRule` with `\markdownRendererThematicBreak`.
